modules/diskq/qdisk.c

```diff
@@ -1393,6 +1392,24 @@ _autodetect_disk_buf_size(QDisk *self)
   return FALSE;
 }
 
+static gboolean
+_ensure_disk_buf_size(QDisk *self)
+{
+  if (self->hdr->disk_buf_size == -1 && !_autodetect_disk_buf_size(self))
+    return FALSE;
```

Comment:
`Failed to load disk-buf-size from disk-queue, and it was not set explicitly` has disappeared. Is this intentional?

Reply:
Yes, actually all of the fail branches of `_autodetect_disk_buf_size()` log an error, so I think there is no need to log another one on the call site.